app/views/pages/security.html.erb

```diff
@@ -0,0 +1,142 @@
+      sign your gems</a> before releasing them to RubyGems.org.
+  </p>
+
+  <h3>Securely downloading Gems</h3>
+  <p>
+    The SHA256hash provided with files on RubyGems.org exists only to provide some
+    download corruption protection. It is not intended to provide any sort of
+    security regarding tampering. Please use GPG signing for that.
+  </p>
+
+  <p>
+    Also, see <a href="https://gist.github.com/bf4/4223e83e8becacfb2a8e">this gist</a>.
+  </p>
+
+  <h2>Reporting RubyGems.org Website Problems</h2>
+  <p>
+    If you're having trouble pushing a gem, or otherwise need help with your
+    RubyGems.org account, please <a href="http://help.rubygems.org">
+      open a new help issue</a>.
+  </p>
+
+  <p>
+    For bugs or other problems with RubyGems.org, please use the
+    <a href="https://github.com/rubygems/rubygems.org">RubyGems.org issue tracker</a>
+    to open a new issue.
+  </p>
+
+  <h2>Disclosure Policy</h2>
+
+  <p>
+    RubyGems and RubyGems.org follow a 5 step disclosure policy:
+  </p>
+
+  <ol>
+    <li>
+      Security report received and is assigned a primary handler. This person
+      will coordinate the fix and release process.
+    </li>
+    <li>
+      Problem is confirmed and, a list of all affected versions is determined.
+      Code is audited to find any potential similar problems.
+    </li>
+    <li>
+      Fixes are prepared for all releases which are still supported. These fixes
+      are not committed to the public repository but rather held locally pending
+      the announcement.
+    </li>
+    <li>
+      A suggested embargo date for this vulnerability is chosen and
+      distros@openwall is notified. This notification will include patches
+      for all versions still under support and a contact address for packagers
+      who need advice back-porting patches to older versions.
+    </li>
+    <li>
+      On the embargo date, the <a href="https://groups.google.com/forum/#!forum/rubygems-developers">
+        rubygems-developers mailing list</a> is sent a copy of the announcement.
+      The changes are pushed to the public repository and new gems released to
+      rubygems. At least 6 hours after the mailing list is notified, a copy of
+      the advisory will be published on the <a href="http://blog.rubygems.org/">
+        RubyGems.org blog</a>.
+    </li>
+  </ol>
+
+  <p>
+    This process can take some time, especially when coordination is required
+    with maintainers of other projects. Every effort will be made to handle
+    the bug in as timely a manner as possible, however it’s important that
+    we follow the release process above to ensure that the disclosure is
+    handled in a consistent manner.
+  </p>
+
+  <h2>Receiving Security Updates</h2>
+
+  <p>
+    The best way to receive all the security announcements is to subscribe
+    to the <a href="https://groups.google.com/forum/#!forum/rubygems-developers">
```

Comment:
Perhaps you want to set up a separate mailing list that's dedicated to security announcements (similar to how RoR has https://groups.google.com/forum/#!forum/rubyonrails-security)? I don't really want to subscribe myself to the main developers list just to get security announcements only. Or just make sure any announcements are cross-posted to the other lists I have mentioned.